src/components/ux-tooltip/ux-tooltip.feature

```diff
@@ -0,0 +1,8 @@
+Feature: ux-tooltip test suite
+
+	Scenario: Loading ux-tooltip
+		Given I have loaded component "ux-tooltip" with use case "dataDriven"
+		When I hover "link"
+		Then there will be an element for "tooltip"
+		When I click "tooltip"
```

Comment:
I know you didn't write it, but same goes for this:

"When I click the "FOO" element"

Might read better?